src/main/java/oss/fosslight/util/PdfUtil.java

```diff
@@ -0,0 +1,186 @@
+package oss.fosslight.util;
+
+import com.itextpdf.html2pdf.HtmlConverter;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.velocity.Template;
+import org.apache.velocity.VelocityContext;
+import org.apache.velocity.app.VelocityEngine;
+import oss.fosslight.CoTopComponent;
+import oss.fosslight.common.CoConstDef;
+import oss.fosslight.common.CommonFunction;
+import oss.fosslight.domain.*;
+import oss.fosslight.repository.CodeMapper;
+import oss.fosslight.repository.MailManagerMapper;
+import oss.fosslight.service.ProjectService;
+import oss.fosslight.service.VulnerabilityService;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.StringWriter;
+import java.io.Writer;
+import java.math.BigDecimal;
+import java.text.SimpleDateFormat;
+import java.util.*;
+
+@Slf4j
+public final class PdfUtil extends CoTopComponent {
+    private static PdfUtil instance;
+
+    private static MailManagerMapper mailManagerMapper;
+    private static VulnerabilityService vulnerabilityService;
+    private static ProjectService projectService;
+    private static CodeMapper codeMapper;
+
+    public static PdfUtil getInstance() {
+        if(instance == null) {
+            instance=new PdfUtil();
+            mailManagerMapper = (MailManagerMapper) getWebappContext().getBean(MailManagerMapper.class);
+            projectService = (ProjectService) getWebappContext().getBean(ProjectService.class);
+            vulnerabilityService=(VulnerabilityService) getWebappContext().getBean(VulnerabilityService.class);
+            codeMapper=(CodeMapper) getWebappContext().getBean(CodeMapper.class);
+        }
+        return instance;
+    }
+    public static ByteArrayInputStream html2pdf(String html) {
+        ByteArrayOutputStream outputStream = new ByteArrayOutputStream();
+        HtmlConverter.convertToPdf(html, outputStream);
+        ByteArrayInputStream inputStream = new ByteArrayInputStream(outputStream.toByteArray());
+        return inputStream;
+    }
+
+    public String htmlToTemplate(String prjId){
+        Map<String,Object> convertData=new HashMap<>();
+        List<OssMaster> oss_Review=new ArrayList<>();
+        List<LicenseMaster> license_Review=new ArrayList<>();
+        List<Vulnerability> vulnerability_Review=new ArrayList<>();
+        List<String> vulnerablity_link=new ArrayList<>();
```

Comment:
You should use camel case.

```suggestion
        List<OssMaster> ossReview = new ArrayList<>();
        List<LicenseMaster> licenseReview = new ArrayList<>();
        List<Vulnerability> vulnerabilityReview = new ArrayList<>();
        List<String> vulnerablityLink = new ArrayList<>();
```

Reply:
Thank you your review!  I modified this code😃